### PR TITLE
Added information about switching SCUs

### PR DESCRIPTION
**Pull request by TSchmiedlechner**

### CODE REVIEW

File: doc/appendix-de-kassensichv/reference-tables/type-of-receipt-ftreceiptcase.md [hunk 1]
```diff
@@ -37,9 +37,11 @@ For Germany (DE), the country code is `0x4445`. Thus, the value of an unknown `f
...
...nly be used after the SCU switch process was initiated in the Portal ([click here for more details](https://link.fiskaltrust.cloud/market-de/scu-switch)). It disconnects the Queue from the currently used, "old" SCU, hence preparing it for connecting another SCU. To finish this process, a _finish SCU switch_ receipt has to be called (see below). After sending this receipt, the Middleware will not contact the SCU/TSE and returns the _ftState_ `0x0000000000000008` until the _finish SCU switch_ receipt is sent.<br /><br />On successful execution, a notification is created which includes relevant data for financial authority notifications.<br /><br />The request is only valid when the charge items block (`ftChargeItems`) and the pay items block (`ftPayItems`) in the `ftReceiptRequest` are empty arrays. <br /><br />This receipt case works **only** with **implicit flow**. Calling it without the `ftReceiptCaseFlag` `0x0000000100000000` ends up in an exception. <br /><br /> Please note that like the _out-of-operation receipt_, this receipt deactivates the currently used TSE by default. To avoid this, please use the respective _ftReceiptCaseFlag_. | [none] <br /> SonstigerVorgang | 1.3.19- |
```
Comment (mijomilicevic) [inline doc/appendix-de-kassensichv/reference-tables/type-of-receipt-ftreceiptcase.md:40]: i think there is a | missing after "initiate scu switch"

File: doc/appendix-de-kassensichv/reference-tables/type-of-receipt-ftreceiptcase.md [hunk 2]
```diff
@@ -37,9 +37,11 @@ For Germany (DE), the country code is `0x4445`. Thus, the value of an unknown `f
...
+| `0x4445000000000018` | **Finish SCU switch**<br /><br />This request is only valid with the same property requirements as a zero-receipt and can only be used after the SCU switch process was initiated in the Portal and an _initiate SCU switch_ receipt was sent ([click here for more details](https://link.fiskaltrust.cloud/market-de/scu-switch)). It connects the Queue to the new SCU, finishing the SCU switch process and making the Middleware operational again.<br /><br />On successful execution, a notification is created which includes relevant data for financial authority notifications.<br /><br />The request is only valid when the charge items block (`ftChargeItems`) and the pay items block (`ftPayItems`) in the `ftReceiptRequest` are empty arrays. <br /><br />This receipt case works **only** with **implicit flow**. Calling it without the `ftReceiptCaseFlag` `0x0000000100000000` ends up in an exception. | [none] <br /> SonstigerVorgang | 1.3.19- |
```
Comment (mijomilicevic) [inline doc/appendix-de-kassensichv/reference-tables/type-of-receipt-ftreceiptcase.md:41]: i think there is a | missing after "finish scu switch"

File: doc/appendix-de-kassensichv/reference-tables/type-of-receipt-ftreceiptcase.md (hunk 1 above)
Comment (TSchmiedlechner) [reply doc/appendix-de-kassensichv/reference-tables/type-of-receipt-ftreceiptcase.md:39]: The problems were caused by the newline, I think :D

File: doc/appendix-de-kassensichv/reference-tables/type-of-receipt-ftreceiptcase.md (hunk 2 above)
Comment (TSchmiedlechner) [reply doc/appendix-de-kassensichv/reference-tables/type-of-receipt-ftreceiptcase.md:40]: See above

**Review by mijomilicevic — APPROVED** [review]

**Pull request merged**